Update virtual-network-bandwidth-testing.md

steveesp opened the pull request:
added link to Read about how [bandwidth is allocated to virtual machines] (virtual-machine-network-throughput.md)

azurecla commented:
Hi __@jeannt__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jeannt). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

steveesp committed:
Update virtual-network-bandwidth-testing.md

added link to Read about how [bandwidth is allocated to virtual machines] (virtual-machine-network-throughput.md)
